azure-pipelines-richnav.yml

```diff
@@ -1,11 +1,10 @@
-# Run on master CI builds
-trigger:
+schedules:
```

Comment:
You also need to include "trigger: none" per https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml

Reply:
Good call thanks!